src/pages/Swap/components/TransactionDetails.tsx

```diff
@@ -150,6 +154,36 @@ export function TransactionDetails({
               />
             </Stack>
           </Stack>
+          <DetailsRow
```

Comment:
should this only be rendered when the swap fees flag is enabled?

Reply:
Very good point